docs/guides/src/main/server/configuration-production.adoc

```diff
@@ -36,4 +36,19 @@ Keycloak runs on top of JGroups and Infinispan, which provide a reliable, high-a
 
 To find out more about using multiple nodes, the different caches and an appropriate stack for your environment, see the <@links.server id="caching"/> guide.
 
+== Stack and address preference
```

Comment:
I'd suggest `Configure Keycloak Server with IPv6 or IPv4`, but this is not a dealbreaker to merge, only a suggest to make things easier to search.

Reply:
Done.